src/PROPOSAL/PROPOSAL/crosssection/parametrization/PhotoPairProduction.h

```diff
@@ -86,5 +95,21 @@ namespace crosssection {
     template <> struct ParametrizationId<PhotoPairKochMotz> {
         static constexpr size_t value = 1000000013;
     };
+
+    // LPM effect object
+    class PhotoPairLPM {
```

Comment:
do we really need this constructor with `mol_density`, `mass_density`, `sum_charge`, `eLPM`? Some variables are not really intuitive.

Reply:
`eLPM` is a computationally expensive calculation that needs to be executed only once, so we should save it.
`mol_density_` and `sum_charge_` are medium variables. Since those are the only two variables we need for the calcluations, we save them to avoid saving the whole `medium` in the PhotoPairLPM object. So I believe these three values are useful to save. Their names are just adopted from what they are called in the `Medium` class (although I agree that the variable names are not always well-chosen, but for the sake of consistency, I would use the same names here).

`mass_density_` is only used in the constructor, so I would agree that we don't need to save them as a private member of the object. But in my opinion, this is the only change I would make to the constructor.